packages/account/PasskeyAccount/passkeyOwnerWebAuthn.ts

```diff
@@ -64,6 +64,28 @@ export class PasskeyOwnerWebAuthn implements PasskeyOwner {
     return signature
   }
 
+  // Avoid exceptions in browserSupportsWebAuthn() to ensure successful webAuthn operations.
+  private supportsWebAuthn(): boolean {
+    let supportsWebAuthn: boolean = false
+    try {
+      supportsWebAuthn = browserSupportsWebAuthn()
+    } catch (e) {
+      if (
+        e instanceof ReferenceError &&
+        e.message.includes("window is not defined")
+      ) {
+        console.warn("An error occurred while checking WebAuthn support:", e)
+      } else {
+        console.error(
+          "An unexpected error occurred while checking WebAuthn support:",
+          e
+        )
+        supportsWebAuthn = false
+      }
+    }
+    return supportsWebAuthn
+  }
```

Comment:
```suggestion
  private isWebAuthnAvailable(): boolean {
    try {
      return browserSupportsWebAuthn()
    } catch (e) {
      console.error(e)
      return false
    } 
  }
```

Reply:
Excellent!
Consider switching console.error() to console.warn() here to differentiate between `expected` and `unexpected` errors, printing specific messages for easier tracking?
```typescript
  private isWebAuthnAvailable(): boolean {
    try {
      return browserSupportsWebAuthn()
    } catch (e) {
      console.warn(`An error occurred while checking WebAuthn support: ${e}`)
      return false
    }
  }
```